src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +14,30 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            session.beginTransaction();
+            session.persist(actor);
+            session.getTransaction().commit();
+        } catch (Exception e) {
+            throw new DataProcessingException("Can't add actor: " + actor, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
```

Comment:
Consider using try-with-resources for the `Session` in the `add` method to ensure it is closed properly, even if an exception occurs. This will make the code cleaner and more robust.